master/reference/public-cloud/aws.md

```diff
@@ -76,3 +81,137 @@ you must configure outgoing NAT on your [Calico IP pool]({{site.baseurl}}/{{page
 AWS will perform outbound NAT on any traffic which has the source address of an EC2 virtual
 machine instance.  By enabling outgoing NAT on your Calico IP pool, Calico will
 NAT any outbound traffic from the containers hosted on the EC2 virtual machine instances.
+
+The following `calicoctl` command will create or modify an IPv4 pool with 
+CIDR 192.168.0.0/16 using IPIP mode `cross-subnet` and enables outgoing NAT.
+Adjust the pool CIDR for your deployment.
+
+```
+$ calicoctl apply -f - << EOF
+apiVersion: v1
+kind: ipPool
+metadata:
+  cidr: 192.168.0.0/16
+spec:
+  ipip:
+    enabled: true
+    mode: cross-subnet
+  nat-outgoing: true
+EOF
+```
+
+## Upgrading from pre-v2.1
+
+If you are planning to use cross-subnet IPIP, your deployment must be running with
+Calico v2.1 or higher.  See [releases page]({{site.baseurl}}/{{page.version}}/releases) 
```

Comment:
hmm.. wouldn't this link to `master` releases page which just says `latest` for all the component version: http://docs.projectcalico.org/master/releases/
I guess it's fine, just thinking out loud